### ISSUE TITLE
student.py : 120行 传入的参数

### ISSUE THREAD

**Issue by viewlei** [HoyTta0/KnowledgeDistillation#3]
student.py ： 120行，在模型训练过程中，get_loss()函数传入的a为1，T为0，这样实现的loss实际上只有 loss(s_logit,label)吧？t-s的损失乘上0之后被忽略了，这里是不是设置的有些出入？
看论文应该是 loss =  a * loss1(s_logit, label) + (1 - a) * loss2(t_logit, s_logit)

谢谢

**Comment by HoyTta0** [comment]
对的，我这里是单独训练学生模型时设置的参数，后面的版本有这一部分的修改，还有输入向量的修改，暂时还没有上传
